crates/transaction-pool/src/pool/parked.rs

```diff
@@ -21,6 +28,8 @@ pub(crate) struct ParkedPool<T: ParkedOrd> {
     submission_id: u64,
     /// _All_ Transactions that are currently inside the pool grouped by their identifier.
     by_id: FnvHashMap<TransactionId, ParkedPoolTransaction<T>>,
+    /// A map between the sender, along with its [TransactionId]s and submission_ids
+    by_sender: FnvHashMap<Address, BTreeSet<EvictionEntry>>,
```

Comment:
what does this keep track of exactly?
all transactions sorted by submission id?

Reply:
removed, but yes, it kept track of txs, sorted by submission id, with an address / sender key